docs/connecting-data/connecting-to-apis-and-external-data/affinity-api.md

```diff
@@ -4,4 +4,156 @@ description: How to Connect DataDistillr to the Affinity API
 
 # Affinity API
 
-Details coming soon!
\ No newline at end of file
+## Creating an Affinity account
+
+You can [contact](https://www.affinity.co/request-demo){target=_blank} Affinity for a free demo and to create an account.
+
+
+??? cost "Cost"
+
+    You can [contact](https://www.affinity.co/request-demo){target=_blank} Affinity for a free demo and pricing.
+
+??? rlimit "Rate Limits"
+
+    Your account plan tier will limit the number of requests each API key can make per month. Current rate limits by plan tier are:
+
+    - Professional tier: 40,000 calls. 
+    - Premium tier: 100,000 calls.
+    - Enterprise tier: unlimited calls.
+
+    All API requests will be halted at 450K per user, per day. Your daily limit will reset the next day at 12AM (midnight) Pacific Time.
+
+## How to Connect DataDistillr to Affinity
+To set up a data source connect for Affinity, you will need to have:
+
+- A [unique name](#name) for your data source connection to be used in queries
+- An [API token](#api-key) generated through your Affinity account
+
+
+### Data Source Form
+To locate the Affinity form, follow the steps in [Connecting Your Data to DataDistillr](../../connecting-to-apis-and-external-data/custom-apis). When you get to the window to choose the data source type, select API as shown below.
```

Comment:
This links to the custom API page rather than the connecting your data to datadistillr page